crates/sparrow-physical/src/lib.rs

```diff
@@ -0,0 +1,19 @@
+#![warn(
```

Comment:
Please especially look at naming / comments to see if the general structure here is clear. I'll flesh these out further as I work on executing them and converting existing plans to physical plans for execution.